packages/account-v2/src/types.ts

```diff
@@ -16,3 +17,26 @@ export type TPOIStatus = Exclude<
 >['identity']['status'];
 
 export type TGetSettingsResponse = ReturnType<typeof useSettings>['data'];
+
+export type TAuthStatusCodes = typeof AUTH_STATUS_CODES[keyof typeof AUTH_STATUS_CODES];
+
+export type TPaymentMethod = keyof ReturnType<typeof getPaymentMethodsConfig>;
+
+export type TPaymentMethodIdentifier = typeof PAYMENT_METHOD_IDENTIFIER[keyof typeof PAYMENT_METHOD_IDENTIFIER];
+
+export type TPaymentMethodInfo = {
+    documentsRequired?: number;
+    icon: JSX.Element;
+    identifier: TPaymentMethodIdentifier | '';
```

Comment:
why do we need empty string as indentifier?
I see we have "none" as indentifier in the config file. what will be the usecase for "